src/Compilers/Core/Portable/DocumentationCommentId.cs

```diff
@@ -615,7 +652,7 @@ public override bool VisitTypeParameter(ITypeParameterSymbol symbol)
                 if (!IsInScope(symbol))
                 {
                     // reference to type parameter not in scope, make explicit scope reference
-                    var declarer = new DeclarationGenerator(_builder);
+                    var declarer = new PrefixAndDeclarationGenerator(_builder);
                     declarer.Visit(symbol.ContainingSymbol);
```

Comment:
>declarer.Visit(symbol.ContainingSymbol); [](http://example.com/codeflow?start=20&length=40)

We don't expect this call to fail? Consider asserting the fact.

Reply:
SGTM.